test/python/algorithms/test_vqe.py

```diff
@@ -296,7 +296,7 @@ def test_with_aer_qasm_snapshot_mode(self):
     @data(
         CG(maxiter=1),
         L_BFGS_B(maxfun=1),
-        P_BFGS(maxfun=1, max_processes=0),
+        P_BFGS(maxfun=1, max_processes=1),
```

Comment:
@ikkoham do you know why this was set to 0? Is 0 processes what we wanted? 🤔

Reply:
As you can see from the `minimize` method, I think this is a non-parallelization option chosen so that the behavior does not change depending on OS in the test.

Reply:
But isn't no-parallelization exactly 1 process? The docstring of `P-BFGS` even says that this must be at least 1. It just didn't raise an error because the value `0` was mistaken for `None` in the input check 🙃